image/rank_order/main.py

```diff
@@ -0,0 +1,25 @@
+# Author: Askar Jaboldinov
```

Comment:
Block would also include license, copyright, dates and description of what the driver/module does.